views/lib/loggedout.hbs

```diff
@@ -5,6 +5,9 @@
 
     {{#content "scripts" mode="append"}}
         <script src="/scripts/loggedout.js" type="text/javascript" defer></script>
+        {{#ifConfig "FEATURE_MATRIX_MESSENGER_ENABLED" true}}
+            <script src="/scripts/chatLoggedout.js" type="text/javascript" defer></script>
```

Comment:
is this also loaded on the login page?

Reply:
No it is currently not, I expected one of `loggedin.hbs` and `loggedout.hbs` to used for every page. But there is also `simple.hbs`. I added the script to remove chat values from local storage the this template too now. It is mainly used for logged out users and  for the `firstLogin.hbs` template.